tools/models.py

```diff
@@ -46,11 +47,11 @@ class Meta:
 
 
 class ToolStates(Catalog):
-    _attrs = "value", "label"
-    none = "none", _("None")
-    unused = "unused", _("Unused")
-    loaned = "loaned", _("Loaned")
-    disabled = "disabled", _("Decommissioned")
+    _attrs = "value", "label", "availability_label", "badge_type"
+    none = "none", _("None"), None, None
+    unused = "unused", _("Unused"), _("Available"), "success"
+    loaned = "loaned", _("Loaned"), _("In Use"), "warning"
+    disabled = "disabled", _("Decommissioned"), _("Retired"), "danger"
```

Comment:
We're still early on. I don't see a reason not to just change these names rather than add a second set of labels.